build-support/ivy/ivysettings.xml

```diff
@@ -41,6 +42,11 @@ Licensed under the Apache License, Version 2.0 (see LICENSE).
            We setup the maven central resolver to require successful pom downloads here. -->
       <ibiblio name="maven-central" m2compatible="true" descriptor="required"/>
 
+      <!-- Custom repo for Kythe jars, as Google don't currently publish them anywhere. -->
```

Comment:
Actually, yea: I think that if the default classpath for the tool is disabled (ie, no built-in default) it should be fine: instead, the tool would be named "kythe" or whatever, and the pantsbuild/pants repo will define the classpath in `BUILD.tools` just like a consumer would.